raiden/transfer/node.py

```diff
@@ -398,10 +398,12 @@ def subdispatch_initiatortask(
             if iteration.new_state:
                 sub_task = InitiatorTask(token_network_identifier, iteration.new_state)
                 chain_state.payment_mapping.setdefault(node_address, PaymentMappingState()).secrethashes_to_task[secrethash] = sub_task
-            elif secrethash in chain_state.payment_mapping[node_address].secrethashes_to_task and not isinstance(state_change, ActionInitInitiatorLight):
+            elif node_address in chain_state.payment_mapping \
+                and secrethash in chain_state.payment_mapping[node_address].secrethashes_to_task \
+                    and not isinstance(state_change, ActionInitInitiatorLight):
                 ## We dont delete the payment task when is a light payment, thats because we need the previous payment task for refunds.
                 ## TODO marcosmartinez7, are the p2p payments being removed?
-                print("Deleted payment task "+ secrethash.hex())
+                print("Deleted payment task {}".format(secrethash.hex()))
```

Comment:
I think using a f-string it's even nicer to read:
`f"Deleted payment task {secrethash.hex()}".`